actors/runtime/types.go

```diff
@@ -6,6 +6,18 @@ import (
 
 // Concrete types associated with the runtime interface.
 
+// Enumeration of network upgrades where actor behaviour can change (without necessarily
+// vendoring and versioning the whole actor codebase).
+type NetworkVersion uint // FIXME move to types
+
+const (
+	NetworkVersion0 = NetworkVersion(iota) // specs-actors v0.9.3
+	NetworkVersion1                        // specs-actors v0.9.7
+	NetworkVersion2                        // specs-actors v2.0.?
+
+	NetworkVersionLatest = NetworkVersion2
+)
```

Comment:
We might want this out in `go-state-types`, right now it's duplicated between branches.